<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="bag_file_path" />
    <arg name="output" />

	<node name="recorder" pkg="rosbag" type="record" args="--lz4 -a -x /cmd_vel -O $(arg bag_file_path)" output="$(arg output)" />

</launch>
